docs/coding-guidelines/javascript.md

```diff
@@ -24,33 +24,36 @@ These rules encourage liberal spacing for improved developer readability. The mi
 
 Don't forget to [remove trailing whitespace](trailing-whitespace.md).
 
+We specify an [EditorConfig](http://editorconfig.org/) configuration and encourage you to install a plugin for your editor to simplify following many of our spacing guidelines.
+
 ## Objects
 
 Object declarations can be made on a single line if they are short (remember the line length guidelines). When an object declaration is too long to fit on one line, there must be one property per line. Property names only need to be quoted if they are reserved words or contain special characters:
 
 ```js
+let map;
+
 // Preferred
-var map = {
-    ready: 9,
-    when: 4,
-    'you are': 15
+map = {
+	ready: 9,
+	when: 4,
+	'you are': 15
 };
 
 // Acceptable for small objects
-var map = { ready: 9, when: 4, 'you are': 15 };
+map = { ready: 9, when: 4, 'you are': 15 };
 
 // Bad
-var map = { ready: 9,
-    when: 4, 'you are': 15 };
-
+map = { ready: 9,
+	when: 4, 'you are': 15 };
```

Comment:
lots of these are `good` vs `bad`, so I think it's reasonable to assume when reading that we're comparing two things that don't exist together